lib/ExecutionEngine/ExecutionEngine.cpp

```diff
@@ -76,6 +76,7 @@ void glow::updateInputPlaceholdersByName(Context &ctx, Module *mod,
 
 void ExecutionEngine::run(Context &ctx) {
   assert(function_ && "No function has been compiled");
+  // TODO call setup functions here not in execute.
```

Comment:
Can you elaborate here?

For example: "We are working on migrating our API from A to B. In the process we will implement C. See task #123456 "